### PR TITLE
feat(quick-trace): Add messages for when quick trace is not present

### PR DESCRIPTION
**Pull request by Zylphrex**
Quick trace in issues can be missing when the corresponding transaction or even
the entire trace is missing. This adds in messages to educate the user about what
happened.

# Screenshots

## Trace and transaction both exists

image

## Trace exists but the transaction is missing

image

## Trace is missing

image

### CODE REVIEW

File: static/app/views/organizationGroupDetails/issueQuickTrace.tsx [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+      }
+
+      return (
+        <StyledAlert type="info" icon={<IconInfo size="sm" />}>
+          <AlertContent>
+            {tct('The [type] for this error cannot be found. To see why, [link].', {
+              type: type === 'missing' ? t('transaction') : t('trace'),
+              link: (
+                <ExternalLink href="https://docs.sentry.io/product/performance/trace-view/#troubleshooting">
+                  go to docs
```
Comment (k-fish) [inline static/app/views/organizationGroupDetails/issueQuickTrace.tsx:119]:
```suggestion
                  {t('go to docs')}
```
?

File: static/app/views/organizationGroupDetails/issueQuickTrace.tsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+      </ErrorBoundary>
+    );
+  }
+}
+
+const LinkContainer = styled('span')`
+  margin-left: ${space(1)};
+  padding-left: ${space(1)};
+  position: relative;
+  font-weight: normal;
```
Comment (k-fish) [inline static/app/views/organizationGroupDetails/issueQuickTrace.tsx:174]: is `font-weight: normal` required?

File: static/app/views/organizationGroupDetails/issueQuickTrace.tsx [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+const StyledAlert = styled(Alert)`
+  margin: 0;
+`;
+
+const AlertContent = styled('div')`
+  display: flex;
+  flex-wrap: wrap;
+
+  @media (min-width: ${p => p.theme.breakpoints[0]}) {
```
Comment (k-fish) [inline static/app/views/organizationGroupDetails/issueQuickTrace.tsx:199]: 👏  Nice breakpoint check

File: static/app/views/organizationGroupDetails/issueQuickTrace.tsx [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+      return null;
+    }
+
+    return (
+      <LinkContainer>
+        <Link
+          to={generateTraceTarget(event, organization)}
+          onClick={() => this.handleTraceLink(organization)}
+        >
+          View Full Trace
```
Comment (k-fish) [inline static/app/views/organizationGroupDetails/issueQuickTrace.tsx:80]:
```suggestion
          {t('View Full Trace')}
```

**Review by k-fish — APPROVED** [review]

File: static/app/views/organizationGroupDetails/issueQuickTrace.tsx (hunk 2 above)
Comment (Zylphrex) [reply static/app/views/organizationGroupDetails/issueQuickTrace.tsx]: No, removed in cd303c6

File: static/app/views/organizationGroupDetails/issueQuickTrace.tsx (hunk 3 above)
Comment (Zylphrex) [reply static/app/views/organizationGroupDetails/issueQuickTrace.tsx:198]: Gotta give credit to @EvanPurkhiser for this one

**Comment by EvanPurkhiser** [comment]
The wording here doesn't feel quite right, should it be 'Read the docs to understand why`

**Comment by Zylphrex** [comment]
> The wording here doesn't feel quite right, should it be 'Read the docs to understand why`

I think you're right, I'll update the wording.

**Review by dashed — APPROVED** [review]

**Pull request merged**